packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -94,6 +103,37 @@ export class Summarizer extends EventEmitter implements ISummarizer {
         this.innerHandle = new SummarizerHandle(this, url, handleContext);
     }
 
+    /**
+     * Creates a Summarizer and its underlying client.
+     * @param loader - the loader that resolves the request
+     * @param url - the URL used to resolve the container
+     * @param options - options used to resolve the container
+     */
+    public static async create(
+        loader: ILoader,
+        url: string): Promise<ISummarizer> {
```

Comment:
Might be worth adding comments around this function and anywhere else that is similar that treatment of URI is different depending on type of loader passed it. One provided and exposed by container runtime is "relative" loader, and thus it knows how to resolve "/", while general ILoader interface does not

Reply:
True.